packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/custom-resource-config/integ.custom-resource-config-logGroup.ts

```diff
@@ -0,0 +1,38 @@
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as logs from 'aws-cdk-lib/aws-logs';
+import * as cdk from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment';
+import { CustomResourceConfig } from 'aws-cdk-lib/custom-resources';
+
+class TestCustomResourceConfig extends cdk.Stack {
```

Comment:
Don't make a new class here. Integration tests follow this convention:

`const stack = new cdk.Stack(this, 'Stack', ...);`

Then you create your resources in the scope of the stack, with

`new s3deploy.BucketDeployment(stack, 's3Deploy', {...});`